models/CreationDate.yaml

```diff
@@ -0,0 +1 @@
+# pk: int
```

Comment:
EOL

but just delete this file.  you are adding a property which does not need another table.